lib/compilers/cmakescript.ts

```diff
@@ -46,6 +46,6 @@ export class CMakeScriptCompiler extends BaseCompiler {
         executeParameters: ExecutableExecutionOptions,
         outputFilename: string,
     ) {
-        executeParameters.args.push('-P', outputFilename);
+        (executeParameters.args as string[]).push('-P', outputFilename);
```

Comment:
https://github.com/compiler-explorer/compiler-explorer/blob/caca3ea6c7045176599e7ab1113a6e85bcda33dc/types/execution/execution.interfaces.ts#L69
isn't this already always `string[]`, I'm a little confused as to why this is needed

Reply:
See below